server/plugins/com/openrsc/server/plugins/quests/members/touristtrap/TouristTrap.java

```diff
@@ -2785,4 +2795,21 @@ private boolean succeedRate(Player p) {
 			return true;
 		}
 	}
-}
+	private boolean playerArmed(Player p) {
+        for (Item item : p.getInventory().getItems()) {
+        for (int robes : allow) {
+            allowed.add(robes);
+        for (int pos : restricted) {
+            wieldPos.add(pos);
```

Comment:
This will add wielded pos multiple times into this array, for every item AND for every robe.